src/droplet.F

```diff
@@ -2201,7 +2201,11 @@ subroutine gauss_newton_2d(diffnorm,dt_nondim,taup_scale,rt_start,rt_zeros,flag,
 
         v1(1) = rt_start(1)
         v1(2) = rt_start(2)
-        fv2 = (/1., 1./)
+         
+        fv2(:) = 1.
+        !JMD-FIXME:  The following line appears to generate the following compiler error:
+        !JMD-FIXME     Internal compiler error. Basic LLVM base data type required 
+        !fv2 = (/1., 1./)
```

Comment:
I thought that I had rewritten all these operations explicitly when I ported BE solver to GPU and encountered a similar issue, but clearly I did not. Thanks for catching this error!